sympy/physics/continuum_mechanics/truss.py

```diff
@@ -0,0 +1,384 @@
+            The label of the starting point/node of the member.
+
+        end: String or Symbol
+            The label of the ending point/node of the member.
+
+        Examples
+        ========
+
+        >>> from sympy.physics.continuum_mechanics.truss import Truss
+        >>> t = Truss()
+        >>> t.add_node('A', 0, 0)
+        >>> t.add_node('B', 3, 0)
+        >>> t.add_node('C', 2, 2)
+        >>> t.add_member('AB', 'A', 'B')
+        >>> t.members
+        [('AB', 'A', 'B')]
+        """
+
+        if start not in self._node_labels or end not in self._node_labels:
+            msg = ("The start and end points of the member must be unique nodes")
+            raise ValueError(msg)
+
+        elif label in self._member_labels:
+            raise ValueError("A member with the same label already exists for the truss")
+
+        elif [start, end] in [[self.members[i][1], self.members[i][2]] for i in range(len(self.members))]:
+            raise ValueError("A member already exists between the two nodes")
+
+        else:
+            self._members.append((label, start, end))
+            self._member_labels.append(label)
+            self._member_nodes[label] = [start, end]
+
+    def remove_member(self, label):
+        """
+        This method removes a member from the given truss.
+
+        Parameters
+        ==========
+        label: String or Symbol
+            The label for the member to be removed.
+
+        Examples
+        ========
+
+        >>> from sympy.physics.continuum_mechanics.truss import Truss
+        >>> t = Truss()
+        >>> t.add_node('A', 0, 0)
+        >>> t.add_node('B', 3, 0)
+        >>> t.add_node('C', 2, 2)
+        >>> t.add_member('AB', 'A', 'B')
+        >>> t.add_member('AC', 'A', 'C')
+        >>> t.add_member('BC', 'B', 'C')
+        >>> t.members
+        [('AB', 'A', 'B'), ('AC', 'A', 'C'), ('BC', 'B', 'C')]
+        >>> t.remove_member('AC')
+        >>> t.members
+        [('AB', 'A', 'B'), ('BC', 'B', 'C')]
+        """
+        if label not in self._member_labels:
+            raise ValueError("No such member exists in the Truss")
+
+        else:
+            self._members.remove((label, self._member_nodes[label][0], self._member_nodes[label][1]))
+            self._member_labels.remove(label)
+            self._member_nodes.pop(label)
+
+    def apply_load(self, location, magnitude, direction):
+        """
+        This method applies an external load at a particular node
+
+        Parameters
+        ==========
+        location: String or Symbol
+            Label of the Node at which load is applied.
+
+        magnitude: Sympifyable
+            Magnitude of the load applied. It must always be positive and any changes in
+            the direction of the load are not reflected here.
+
+        direction: Sympifyable
+            The angle, in degrees, that the load vector makes with the horizontal. It takes
+            the values 0 to 360, inclusive.
```

Comment:
Add the direction here.
` the load vector makes with the horizontal in the counterclockwise direction`